### DISCUSSION TITLE
Any logging/graphing options?

### DISCUSSION THREAD

**Discussion by dwblaikie** [Rem0o/FanControl.Releases#1274] [Ideas]
I'd love to collect temp values over hours/days (I leave my computer on while I'm away for extended periods (Cookie Clicker, of all things - we all have our vices ;) ) - and just setup my first water cooling loop, so would like to keep a closer eye on it) - possibly with extra details about fan speed and maybe even recording  FanControl's choices to better understand/see how it's responding to conditions.

Does FanControl support logging? Any interest in adding it? I might be able to contribute patches, if it's of interest.

**Comment by Rem0o** [comment]
FanControl has no monitoring features, as intended.  None are planned.

**Reply by dwblaikie** [reply]
Thanks for the quick reply!

Is the plugin interface flexible enough to enable this, or should I look for other programs/write my own/etc?

**Reply by Impact123** [reply]
I'd suggest you use software like `LibreHardwareMonitor` or `HWiNFO` which have the ability to log sensors.
